js/google_ima.js

```diff
@@ -1034,6 +1034,7 @@ require("../html5-common/js/utils/utils.js");
         //These are required by Google for tracking purposes.
         google.ima.settings.setPlayerVersion(PLUGIN_VERSION);
         google.ima.settings.setPlayerType(PLAYER_TYPE);
+        google.ima.settings.setLocale(OO.getLocale());
```

Comment:
Yes, I tested what would happen if capital letters were passed to IMA; it works. It looks like they probably lowercase the input to setLocale().

![image](https://cloud.githubusercontent.com/assets/15880169/22672633/05636136-ec8a-11e6-9909-bd22313d27c5.png)